packages/react-components/src/entities/EventSidebar/details/Summaries.js

```diff
@@ -83,7 +83,7 @@ export function Summaries({ event, setActiveEvent, addToSearch, addEventTypeToSe
             name: nodes[i],
             isSelected: false,
             count: hierarchy.count,
-            isClickable: true,
+            isClickable: hierarchy.count == 1 ? true:false,
```

Comment:
unnecessary ternary as far as I can see? `hierarchy.count === 1` should suffice?

Reply:
Nice in deed